R/modelplot.R

```diff
@@ -225,32 +225,24 @@ modelplot <- function(models,
   if (!is.null(conf_level)) {
     if (length(unique(dat$model)) == 1) {
       args_list <- list(
-        mapping = ggplot2::aes(y = model, x = estimate, 
-                               xmin = conf.low, xmax = conf.high)
+        mapping = ggplot2::aes(y = term, x = estimate, xmin = conf.low, xmax = conf.high)
```

Comment:
@florence-laflamme I found a couple minor bugs like this here, where you had swapped `y=model` and `y=term`. Both are relevant, but it's important where they are placed in the logic. No big deal. Should be fixed now.